### PR TITLE
Update to Golang 1.13.9

### PR DESCRIPTION
**Pull request by justaugustus**
**What type of PR is this?**
/kind feature cleanup
/area dependency

**What this PR does / why we need it**:
- deps: Update to Golang 1.13.9
- build: Remove kube-cross image building

Accompanying kube-cross bump: https://github.com/kubernetes/release/pull/1200

/assign @BenTheElder @liggitt @cblecker @dims @listx
/milestone v1.18
cc: @kubernetes/release-team @kubernetes/release-engineering 
ref: [quarterly updates](https://github.com/kubernetes/release/issues/1134)

**Does this PR introduce a user-facing change?**:

```release-note
- deps: Update to Golang 1.13.9
- build: Remove kube-cross image building
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by justaugustus** [comment]
(Dropped the staging image commit (2ae4a0ed0e146e435984171cac4dffed415df064) now that the [kube-cross:v1.13.9-2 image has been promoted](https://github.com/kubernetes/k8s.io/pull/684))

**Comment by justaugustus** [comment]
This PR is currently blocked on tests failing due to bazel.
I've provided details here: https://github.com/kubernetes/kubernetes/issues/88553#issuecomment-602411809

**Comment by ixdy** [comment]
An alternative to updating rules_go is https://github.com/ixdy/kubernetes/commit/2a189ad34ee611a02a4bc1cda12deddcf1c15a44.

**Comment by justaugustus** [comment]
@ixdy -- Thanks so much! I've squashed that in as b6179ac.

**Comment by justaugustus** [comment]
Looks like most of the tests are starting to pass, so I'm going to apply @ixdy's fix on the 1.14.1 PR (https://github.com/kubernetes/kubernetes/issues/88638) as well.

**Comment by justaugustus** [comment]
Added 979e3e0dd529bbcd0f0cdef22fc7f82f66ae48ce:
```
build/dependencies: Remove bazel WORKSPACE go_version check
    
When supplying the go_download_sdk instead of updating rules_go, the
go_version is removed, so we shouldn't check for it anymore in
hack/verify-external-dependencies-version.sh.

Without this, `pull-kubernetes-verify` jobs will fail.
```

File: build/build-image/cross/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v1.13.8-1
+v1.13.9-2
```
Comment (liggitt) [inline build/build-image/cross/VERSION]: this remains but the rest of the image is gone?

Comment (justaugustus) [reply build/build-image/cross/VERSION]: @liggitt -- Yep. There are still a [few things that depend on it](https://cs.k8s.io/?q=cross%2FVERSION&i=nope&files=&repos=) being in that location.

So basically:
- production version: https://github.com/kubernetes/kubernetes/blob/master/build/build-image/cross/VERSION
- prerelease version: https://github.com/kubernetes/release/blob/master/images/build/cross/VERSION

I didn't want to introduce changing the references to this PR, but I am thinking about the best way to do it for a follow-up.

Comment (liggitt) [reply build/build-image/cross/VERSION]: ok, as long as its on your radar

File: build/root/WORKSPACE [hunk 2]
```diff
@@ -48,14 +48,31 @@ http_archive(
...
+        "freebsd_amd64": ("go1.13.9.freebsd-amd64.tar.gz", "87716246da52c193226df44031aaf45e45ebfc23e01bdc845311c1b560e76e2b"),
+        "linux_386": ("go1.13.9.linux-386.tar.gz", "a2744aa2ddc68d888e9f65c2cbe4c8b527b139688ce232ead90dc2961f8d51a8"),
+        "linux_amd64": ("go1.13.9.linux-amd64.tar.gz", "f4ad8180dd0aaf7d7cda7e2b0a2bf27e84131320896d376549a7d849ecf237d7"),
+        "linux_arm64": ("go1.13.9.linux-arm64.tar.gz", "b53cb466d7986e5e17a3d4c196bc95df08a35968eced5efd7e128387a246c46e"),
+        "linux_arm": ("go1.13.9.linux-armv6l.tar.gz", "a3c2941a1fde8692514ece7e2180a0e3ca70609f52756a66bc0ab68c63572361"),
+        "linux_ppc64le": ("go1.13.9.linux-ppc64le.tar.gz", "90beb01962202f332be0a7c8dad2db3d30242759ba863db3f36c45d241940efc"),
+        "linux_s390x": ("go1.13.9.linux-s390x.tar.gz", "a40949aaf55912b06df8fda511c33fde3e52d377706bdc095332652c1ad225e3"),
+        "windows_386": ("go1.13.9.windows-386.zip", "e22406377448f1aea2dd1517327e5ae452d826c0c7624b3511d5af510c57b69a"),
+        "windows_amd64": ("go1.13.9.windows-amd64.zip", "cf066aabdf4d83c251aaace14b57a35aafffd1fa67d54d907f27fb31e470a135"),
+    },
```
Comment (liggitt) [inline build/root/WORKSPACE]: deep magic from the dawn of time

**Comment by liggitt** [comment]
go bits look good. I'll take @ixdy's word on the bazel bits from https://github.com/ixdy/kubernetes/commit/2a189ad34ee611a02a4bc1cda12deddcf1c15a44

/lgtm
/approve

File: build/root/WORKSPACE (hunk 2 above)
Comment (ixdy) [reply build/root/WORKSPACE]: it's a little bit of https://github.com/kubernetes/kubernetes/commit/efb21a54f19d6f64f3272df5d3048bd68360dcea mixed with https://github.com/bazelbuild/rules_go/blob/master/go/private/sdk_list.bzl.

**Comment by justaugustus** [comment]
Cherry picks opened:
- 1.18: https://github.com/kubernetes/kubernetes/pull/89398
- 1.17: https://github.com/kubernetes/kubernetes/pull/89399
- 1.16: https://github.com/kubernetes/kubernetes/pull/89400

**Pull request merged**